src/components/molecules/plan/plan.scss

```
@@ -0,0 +1,64 @@
+@use "../../variables" as *;
+
+.plan-list {
+  list-style: none;
+  text-align: center;
+  margin-bottom: 24px;
+  @media screen and (min-width: $breakpoint-md) {
+    margin-bottom: 0;
+  }
+  padding-inline-start: 0;
+  &.odd {
+    border: solid 1px $gray_gainsboro;
+    & > li:first-child {
+      background-color: $orange_zorba;
+      padding: 20px;
+    }
+  }
+  &.even {
+    border: solid 5px $orange_bistre;
+    & > li:first-child {
+      background-color: $orange_bistre;
+      padding: 15.5px;
+    }
+    & > li:last-child {
+      padding: 30px;
+      @media screen and (min-width: $breakpoint-md) {
+        padding: 20px;
+      }
+    }
+  }
+  & > *:not(:last-child) {
+    border-bottom: solid 1px $gray_gainsboro;
+  }
```

```suggestion
  & > li:not(:last-child) {
    border-bottom: solid 1px $gray_gainsboro;
  }
```
これでいける気がします。（総称セレクタは影響範囲の予測が難しいのでなるべく使いたくないかと思います。）